assets/src/setup/components/options-context-provider.js

```diff
@@ -126,5 +130,5 @@ export function OptionsContextProvider( { children, optionsRestEndpoint } ) {
 
 OptionsContextProvider.propTypes = {
 	children: PropTypes.any,
-	optionsRestEndpoint: PropTypes.string.isRequired,
+	optionsRestEndpoint: PropTypes.string,
```

Comment:
This was for unit testing purposes, but I've revised the approach; rather than changing the prop here (it should be required), in the test I've just mocked an OptionsContextProvider that doesn't fetch.